content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -67,44 +124,3 @@ on the products' bill control policy.
 +--------------------+------------------------------------------------+-------------------------+
 | *Fully Billed*     | All/some products received; draft bill created |    Draft bill created   |
 +--------------------+------------------------------------------------+-------------------------+
```

Comment:
Table formatting needs update.

1. `:guilabel:` only
```suggestion
+------------------------------+--------------------------------------------------------------------------+
| :guilabel:`Billing Status`   |                              **Conditions**                              |
|                              +------------------------------------------------+-------------------------+
|                              |            *On received quantities*            | *On ordered quantities* |
+------------------------------+------------------------------------------------+-------------------------+
| :guilabel:`Nothing to Bill`  |       PO confirmed; no products received       |     *Not applicable*    |
+------------------------------+------------------------------------------------+-------------------------+
| :guilabel:`Waiting Bills`    |  All/some products received; bill not created  |       PO confirmed      |
+------------------------------+------------------------------------------------+-------------------------+
| :guilabel:`Fully Billed`     | All/some products received; draft bill created |    Draft bill created   |
+------------------------------+------------------------------------------------+-------------------------+
```

OR 

2. Mix of `:guilabel:` and `pre` formatting also works since `pre` indicates a UI field's value.
```suggestion
+------------------------------+--------------------------------------------------------------------------+
| :guilabel:`Billing Status`   |                              **Conditions**                              |
|                              +------------------------------------------------+-------------------------+
|                              |            *On received quantities*            | *On ordered quantities* |
+------------------------------+------------------------------------------------+-------------------------+
|      `Nothing to Bill`       |       PO confirmed; no products received       |     *Not applicable*    |
+------------------------------+------------------------------------------------+-------------------------+
|      `Waiting Bills`         |  All/some products received; bill not created  |       PO confirmed      |
+------------------------------+------------------------------------------------+-------------------------+
|      `Fully Billed`          | All/some products received; draft bill created |    Draft bill created   |
+------------------------------+------------------------------------------------+-------------------------+
```

Reply:
I went with the :guilabel: only table format.